desidiff/nb/desidiff.ipynb

```diff
@@ -196,15 +210,50 @@
     "            plt.show()\n",
     "            count=count+1\n",
     "            \n",
+    "            #SkyPortal functionality begins:\n",
+    "            #Combine the wavelengths into one \n",
+    "            wavelengths = []\n",
+    "            for b in difflux.keys():\n",
+    "                wavelengths += difwave[b].tolist()\n",
```

Comment:
I don't think this renders properly.  Different channels have overlapping wavelengths.